apps/extension/src/core/views/ActivityItem.tsx

```diff
@@ -1,13 +1,61 @@
+import {
+  MediaMimeType,
+  isMediaOutput,
+  isPromptInput,
+  mimeTypeToExtension,
+  isMediaMimeType
+} from "window.ai"
+
 import { Logo } from "~core/components/pure/Logo"
 import { Text } from "~core/components/pure/Text"
 import { originManager } from "~core/managers/origin"
 import type { Transaction } from "~core/managers/transaction"
 import { transactionManager } from "~core/managers/transaction"
 import { formatDate } from "~core/utils/utils"
 
+function createMediaDownloadLinks(transaction: Transaction) {
+  if (!transaction.input || !transaction.outputs) {
```

Comment:
there should always be an `input`, per the type

Reply:
so this should just check `outputs`

Reply:
fixed